[2022-09-04T16:36:26.283+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime manual__2022-09-04T16:36:25.222677+00:00 [queued]>
[2022-09-04T16:36:26.289+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime manual__2022-09-04T16:36:25.222677+00:00 [queued]>
[2022-09-04T16:36:26.291+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-04T16:36:26.292+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-04T16:36:26.293+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-04T16:36:26.300+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): get_datetime> on 2022-09-04 16:36:25.222677+00:00
[2022-09-04T16:36:26.305+0000] {standard_task_runner.py:52} INFO - Started process 4021 to run task
[2022-09-04T16:36:26.308+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_***_dag', 'get_datetime', 'manual__2022-09-04T16:36:25.222677+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp0n7lwlqj', '--error-file', '/tmp/tmpphuu67ry']
[2022-09-04T16:36:26.310+0000] {standard_task_runner.py:80} INFO - Job 97: Subtask get_datetime
[2022-09-04T16:36:26.353+0000] {task_command.py:371} INFO - Running <TaskInstance: first_airflow_dag.get_datetime manual__2022-09-04T16:36:25.222677+00:00 [running]> on host c56acb3a0493
[2022-09-04T16:36:26.393+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_***_dag
AIRFLOW_CTX_TASK_ID=get_datetime
AIRFLOW_CTX_EXECUTION_DATE=2022-09-04T16:36:25.222677+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-04T16:36:25.222677+00:00
[2022-09-04T16:36:26.395+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-09-04T16:36:26.396+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'date']
[2022-09-04T16:36:26.405+0000] {subprocess.py:85} INFO - Output:
[2022-09-04T16:36:26.407+0000] {subprocess.py:92} INFO - Sun Sep  4 16:36:26 UTC 2022
[2022-09-04T16:36:26.409+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2022-09-04T16:36:26.428+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=first_***_dag, task_id=get_datetime, execution_date=20220904T163625, start_date=20220904T163626, end_date=20220904T163626
[2022-09-04T16:36:26.480+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-04T16:36:26.507+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
